### PR TITLE
Update to OpenStudio 3.4.0

### PR DESCRIPTION
**Pull request by ThibaultMarzullo**
**Changes for the OS 3.3.0 update:**

1. The gemspec was updated to use:

- openstudio-common-measures 0.5.0
- openstudio-ee 0.5.0
- openstudio-extension 0.5.1
- openstudio-model-articulation 0.5.0
- openstudio-standards 0.2.15

2. The in.osm file was updated to OS 3.3.0
3. The in.idf file was updated to EP 9.6

**Extras**
In workflow_maker_spec, one test consists of inserting an EnergyPlus measure into the workflow. The measure that was used is broken (_ModifyEnergyPlusCoilCoolingDXSingleSpeedObjects_), it was replaced by another EnergyPlus measure (_AddSimplePvToShadingSurfacesByType_).

**Notable change to EE measures used in BSync tests**

The EE-measures-gem used to include measures that were used in tests, but as of v0.5.0, 13 of these measures are missing or unsupported. There were a total of 30 measures used in tests, now we have 17.

1. Removed the measures from phase_zero_base.osw
2. Removed the measures and their scenarios from building_151.xml and building151_n1.xml which were used for testing
3. Updated spec files to expect less scenarios since we now have less measures. Instead of 30 scenarios, we now expect 17 scenarios in the following specs: translator_spec, facility_spec, spec_helper

The complete list of measures that were removed is:
- Replace with ENERGY STAR rated
- improve_simple_glazing_by_percentage
- set_water_heater_efficiency_heat_lossand_peak_water_flow_rate
- replace_simple_glazing
- replace_hvac_with_gshp_and_doas
- vr_fwith_doas
- add_apszhp_to_each_zone
- ReplaceFanTotalEfficiency
- ImproveFanTotalEfficiencybyPercentage
- add_energy_recovery_ventilator
- set_water_heater_efficiency_heat_lossand_peak_water_flow_rate
- set_water_heater_efficiency_heat_lossand_peak_water_flow_rate
- reduce_water_use_by_percentage

### CODE REVIEW

**Comment by ThibaultMarzullo** [comment]
Edit: I forgot to update the CI file to use the nrel/openstudio:330 docker container. It's fixed now.

The CI tests fail, in the setup section we have the below log. Any idea what this is?
```Bundler found conflicting requirements for the Ruby� version:
  In Gemfile:
    buildingsync was resolved to 0.2.1, which depends on
      Ruby� (~> 2.7.0)

  Current Ruby� version:
    Ruby� (= 2.5.1)
Error: Process completed with exit code 6.

**Comment by ThibaultMarzullo** [comment]
This new commit relates to:
- the update to OS 3.4.0 [Project Tracker #132](https://github.com/BuildingSync/project-tracker/issues/132)
Additionally, this PR solves the update to OS 3.2.0 [Project Tracker #84](https://github.com/BuildingSync/project-tracker/issues/84)

File: lib/buildingsync/makers/phase_zero_base.osw [hunk 1]
```diff
@@ -20,11 +20,6 @@
 			"arguments": {
 				"__SKIP__": true
 			}
-		}, {
```
Comment (nllong) [inline lib/buildingsync/makers/phase_zero_base.osw:23]: nice, removing this should appease @DavidGoldwasser ... haha.

File: lib/buildingsync/version.rb [hunk 2]
```diff
@@ -41,5 +41,5 @@
 # provides features for reading BldgSync XML files, generating baseline models, creating scenario workflows, running simulations and adding simulation results to the BldgSync file
 module BuildingSync
   # version of the BuildingSync gem
-  VERSION = '0.2.1'
+  VERSION = '0.2.2'
```
Comment (nllong) [inline lib/buildingsync/version.rb:44]: I think we should bump this to 0.3.0? thoughts on why not to?

File: spec/tests/workflow_maker_spec.rb [hunk 3]
```diff
@@ -244,29 +244,30 @@
...
-      ['EnergyPlusMeasure', 'ModifyEnergyPlusCoilCoolingDXSingleSpeedObjects', 1, 27, {
-        'ratedTotalCoolingCapacity' => 999.9,
-        'ratedCOP' => 0.99,
-        'ratedAirFlowRate' => 0.999,
-        'condensateRemovalStart' => 9.999,
-        'evapLatentRatio' => 0.0999,
-        'latentCapTimeConstant' => 4.0
-      }],
-      ['ReportingMeasure', 'openstudio_results', 0, 29, nil],
+      #['EnergyPlusMeasure', 'ModifyEnergyPlusCoilCoolingDXSingleSpeedObjects', 0, 15, {
```
Comment (nllong) [inline spec/tests/workflow_maker_spec.rb:247]: i'd suggest just removing the commented lines.

**Review by nllong — APPROVED** [review]
looks great!

**Comment by nllong** [comment]
I also think this is upgrading to `3.4.0` not `3.3.0` as stated in the PR name.

**Comment by DavidGoldwasser** [comment]
@ThibaultMarzullo should `building_sync_to_openstudio` measure from this branch be in this? https://github.com/BuildingSync/BuildingSync-gem/tree/update-os-measure/lib/measures/building_sync_to_openstudio

**Comment by laurenliz22** [comment]
https://github.com/BuildingSync/BuildingSync-gem/pull/139

File: lib/buildingsync/version.rb (hunk 2 above)
Comment (laurenliz22) [reply lib/buildingsync/version.rb:44]: I think that makes sense, this is a major change.  It shouldn't break anything.

**Comment by nllong** [comment]
This PR looks to now be behind #140. I recommend cleaning this up and merging down. Then pulling into #140 (maybe with `--rebase`).

**Comment by nllong** [comment]
🥳

**Pull request merged**